docs/gui/usermanagement_en.md

```diff
@@ -1,209 +1,245 @@
 
-In User Management, you will find predefined groups under “Groups”. These groups have special permissions in the Cloudogu EcoSystem dogus.
+## Permission concept in the Cloudogu EcoSystem
 
-**Manager group**
+The permission concept of the Cloudogu EcoSystem is based on a **central user management** and a **decentralized permission configuration**: Accounts and groups can be stored in the **User Management**. These are propagated to the other Dogus of the Cloudogu EcoSystem, allowing you to assign permissions for groups or individual users decentralized in each Dogu.
 
-Members of this group have full access to the “User Management” of the Cloudogu EcoSystem.
-This gives users the authorization to create and manage additional users on the one hand and groups on the other.
-Beyond that, no further authorizations are associated with the “Manager” group.
+### User Management
 
-You can change the “Manager” group to be used by editing the configuration of the Cloudogu EcoSystem in the entry
-`/config/_global/manager_group` to set the desired group.
+The **User Management** is used to manage accounts and groups. In doing so, the **User Management** uses an internal *LDAP* as directory service.
 
-```shell
-etcdctl set /config/_global/manager_group newManagerGroup
-```
+In addition to using the **User Management** provided by us, you have the option of using an **external directory service** for the Cloudogu EcoSystem. In this case, user management would **not** be done via the Dogu **User Management** presented here, but via the external directory service you have connected (such as an external *LDAP* or *Active Directory*).
 
-The “User Management” dogu must then be restarted for the change to take effect.
+### Permission concepts of the Dogus
 
-**Admin group**
+You can create the accounts for users of the Cloudogu EcoSystem centrally in the **User Management**. To simplify the permission configuration, you can create groups for different user groups. A user can belong to more than one group. A group can have more than one member.
 
-Members of this group have administrative privileges in **all** dogus of the Cloudogu EcoSystem, e.g. Jenkins, Redmine or SCM-Manager. 
-permissions. This means that users can use the administrative functions in the individual dogus 
-and thus, for example, install plug-ins or make application settings.
+Accounts and groups are **synchronized** with the Dogus, meaning in each Dogu you will find the accounts and groups created in the **User Management**. 
 
-The  “Backup & Restore” dogu contains only administrative functions and does not use a separate role concept.
-Thus, only users who are members of the “Admin” group have access to the “Backup & Restore” dogu.
+Since Dogus may be systems developed outside the Cloudogu EcoSystem, the **permission concept in the Dogus may differ** - as you can see in the following diagram exemplary.
```

Comment:
```suggestion
Since Dogus may be applications developed outside of the Cloudogu EcoSystem, the **permission concept in the Dogus may differ** - as you can see in the following diagram exemplary.
```